kpatch-build/kpatch-cc

```diff
@@ -79,6 +79,24 @@ elif [[ "$TOOLCHAINCMD" =~ ^(.*-)?ld || "$TOOLCHAINCMD" =~ ^(.*-)?ld.lld ]] ; th
 					args+=(--warn-unresolved-symbols)
 					break
 					;;
+				*/.tmp_*.o)
+					# .tmp_*.o is used for single file modules.
+					# See "cmd_ld_single_m" in scripts/Makefile.build.
+					if [[ $KPATCH_CC_HANDLE_LTO -ne 0 ]] ; then
```

Comment:
I think "[[ -n $KPATCH_CC_LTO -eq 1 ]]" would be clearer since we never actually initialize the variable to 0 (and also it avoids the double negative check)